plugins/modules/dns_record_info.py

```diff
@@ -0,0 +1,406 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Infoblox Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: dns_record_info
+short_description: Manage Record
+description:
+    - Manage Record
+version_added: 2.0.0
+author: Infoblox Inc. (@infobloxopen)
+options:
+    id:
+        description:
+            - ID of the object
+        type: str
+        required: false
+    filters:
+        description:
+            - Filter dict to filter objects
+        type: dict
+        required: false
+    filter_query:
+        description:
+            - Filter query to filter objects
+        type: str
+        required: false
+    inherit:
+        description:
+            - Return inheritance information
+        type: str
+        required: false
+        choices:
+            - full
+            - partial
+            - none
+        default: full
+    tag_filters:
+        description:
+            - Filter dict to filter objects by tags
+        type: dict
+        required: false
+    tag_filter_query:
+        description:
+            - Filter query to filter objects by tags
+        type: str
+        required: false
+
+extends_documentation_fragment:
+    - infoblox.bloxone.common
+"""  # noqa: E501
+
+EXAMPLES = r"""
+  - name: Get DNS A Record information by ID
+    infoblox.bloxone.a_record_info:
+      id: "{{ a_record_id }}"
+
+  - name: Get DNS A Record information by filters (e.g., name_in_zone)
+    infoblox.bloxone.a_record_info:
+      filters:
+        name_in_zone: "example_a_record"
+        type: "A"
+
+  - name: Get DNS A Record information by raw filter query
+    infoblox.bloxone.a_record_info:
+      filter_query: "name_in_zone=='example_a_record' and type=='A'"
+
+  - name: Get DNS A Record information by filters (zone)
+    infoblox.bloxone.a_record_info:
+      filters:
+        zone: "example_zone_id"
+        type: "A"
+
+  - name: Get DNS A Record information by filter query for zone
+    infoblox.bloxone.a_record_info:
+      filter_query: "zone=='example_zone_id' and type=='A'"
+
+  - name: Get DNS A Record information by filters (absolute_name_spec)
+    infoblox.bloxone.a_record_info:
+      filters:
+        absolute_name_spec: "example_a_record.example.com"
```

Comment:
```suggestion
        absolute_name_spec: "example_a_record.{{ auth_zone.fqdn}}"
```
This can be added for more clarity , although its not necessary to create auth zone , we can just keep this as a placeholder.